stock_picking_batch_print_pickings/readme/USAGE.md

```diff
@@ -0,0 +1,13 @@
+To use this module, you need to:
+
+1. Go to Inventory > Configuration > Operation Types.
+2. Choose one already created or create new operation type.
+3. Check print pickings.
+4. Fill in the number of copies you want.
+5. Go to Inventory > Operations > Transfers.
+6. Create some new transfer with operation type configured before.
+7. Go to Inventory > Operations > Batch Transfers.
+8. Create new batch and add transfers created before.
```

Comment:
**issue:** without saving, the button you need to click on next step doesn't appear.

```suggestion
8. Create new batch, add transfers created before and save.
```